src/test/scala/mesosphere/marathon/integration/setup/ZookeeperServer.scala

```diff
@@ -1,19 +1,18 @@
 package mesosphere.marathon.integration.setup
 
 import com.typesafe.scalalogging.StrictLogging
-import mesosphere.marathon.core.storage.store.impl.zk.{ NoRetryPolicy, RichCuratorFramework }
+import mesosphere.marathon.core.storage.store.impl.zk.{NoRetryPolicy, RichCuratorFramework}
```

Comment:
Thanks for noting. The branch should have only changes to the Jenkinsfile. I'll clean that up.